tools/cli/flags.go

```diff
@@ -335,9 +335,9 @@ func getFlagsForStart() []cli.Flag {
 			Usage: "Configure if the same workflow Id is allowed for use in new workflow execution. " +
 				"Options: AllowDuplicate, AllowDuplicateFailedOnly, RejectDuplicate",
 		},
-		cli.StringFlag{
+		cli.StringSliceFlag{
 			Name:  FlagInputWithAlias,
-			Usage: "Optional input for the workflow, in JSON format. If there are multiple parameters, concatenate them and separate by space.",
+			Usage: "Optional input for the workflow, in JSON format. If there are multiple parameters, pass each as a separate input flag.",
```

Comment:
[Nit] do we need to mention anything about the input flag ordering corresponding to the input on the workflow params, or is this implicit enough for a user to get?

Reply:
It's good point. I would add something like "Order of input flags must be the same as parameters order."

Reply:
i think they will expect the ordering to matter, i would rather keep the help page relatively simple, focusing on the main information. However we could add this to our docs. I've added an item to Cully regarding the docs

Reply:
https://www.notion.so/f03f62f4cefd498fa8caaae2bb834eb4?v=7466982797634db691c9448414f1109a&p=90335cb03b0341319d97160b8281d3ea